README.md

```diff
@@ -72,10 +72,11 @@ Place the binary in a folder, shined upon by your `$PATH`.
  * For macos & linux, that's typically `/usr/local/bin/`
 
 # Quickstart
-You will need a `.todocheck.yaml` configuration file in the root of your project first.  
+For public github/gitlab projects it will detect the issue tracker from your git configuration.
+Otherwise you will need a `.todocheck.yaml` configuration file in the root of your project first.  
```

Comment:
This line assumes some context although it's in the start of the `Quickstart` section. This section will typically be read by newcomers to the tool.

For example, it already mentions an issue tracker when it isn't already mentioned that todocheck needs a way to integrate with one.

I suggest restructuring this part in the following way:
```
First, you need to configure `todocheck`'s integration with your issue tracker.
This is done by creating a `.todocheck.yaml` file in the root of your project.

For public github or gitlab repositories, you need not specify a config file, as long as you are using git, as `todocheck` will interpret your issue tracker from the git configuration.

Alternatively, the explicit configuration for a public github repository would look like this:
(config follows)
```